### ISSUE TITLE
Feature Request: 2-way sync between GBA & GBA 2P cores

### ISSUE THREAD

**Issue by KremlingKuthroat19** [MiSTer-devel/GBA_MiSTer#131]
I have a symlink setup in the Game Boy core where I've symlinked the Game Boy, Game Boy 2P and Super Game Boy cores to use the game and save folders of the standalone Game Boy core. If I make progress on the Game Boy core and save it, I can continue where I've left off on the Game Boy 2P or Super Game Boy core, and any combination of the three cores. It's brilliant. A true 3-way sync!

I've set up a symlink for the games and saves of the GBA and GBA 2P core. Although it works well, the only issue is that there isn't a 2-way sync, just a 1-way sync for save files. Using DKC as an example, I completed the first level and saved on the stock GBA core. The save file appears on the GBA 2P core and I completed the second level and saved. The save remains until I play the standalone GBA core and save and then that save overwrites the GBA 2P core's save file.

TLDR; there's a 1-way sync for save files between the GBA and GBA 2P cores. The standalone GBA core's saves takes priority over the GBA 2P core. Is there a way to fix this? It works flawlessly in the Game Boy cores so I'm hoping there's a solution :)

**Comment by RobertPeip** [comment]
I don't see why they should behave different. The cores are doing the same: they write a .sav file with the internal saveram contents.

Also i don't understand it. You write that GBA overwrites GBA2P, but isn't that what you wanted?

**Comment by RobertPeip** [comment]
fixed with comfirmation from Kremling
